doc/operations/start_network.rst

```diff
@@ -103,33 +103,33 @@ The following diagram summarises the steps that operators and members should fol
             Node 0-->>-Node 1: "Pending" state
         end
 
-        Operators->>+Node 1: Poll /node/state for "PartOfNetwork"
+        Operators->>+Node 1: Poll /node/state for "PartOfNetwork" (using self-signed certificate as CA)
```

Comment:
As identity, not CA?

Reply:
Hmm, `curl` only has the `--cacert` option, Python `requests` calls it the `REQUESTS_CA_BUNDLE`, hence the name here.

Reply:
Ok